src/index.js

```diff
@@ -1,7 +1,16 @@
 import low from 'lowlight'
 import visit from 'unist-util-visit'
 
-export default function attacher({include, exclude, prefix} = {}) {
+export default function attacher({include, exclude, prefix, languages} = {}) {
+
+  // register custom languages
+  if (languages) {
+    // eslint-disable-next-line guard-for-in
```

Comment:
ideally this should be addressed, not ignored.
https://eslint.org/docs/rules/guard-for-in offers some potential solutions

Reply:
Put a:

```js
const own = {}.hasOwnProperty
```

somewhere at the top and then in the loop wrap the body in:

```
if (own.call(languages, name)) {
  // ...
}
```